.github/workflows/ci_test.yml

```diff
@@ -100,6 +100,16 @@ jobs:
         run: TF_ADDONS_NO_BUILD=1 pip install --no-deps -e .
       - name: Run type check
         run: python tools/ci_build/verify/check_typing_info.py
+  case-insensitive-filesystem:
+    name: Works on case insensitive FS
```

Comment:
Mind just renaming this slightly for clarity in the CI test names that are displayed. `Verify python filenames work` or `Verify python filenames work on case sensitive FS`

I believe it should be `case sensitive` btw so might need to rework the comment in the `check_file_name` file

Reply:
Are you sure it's case sensitive? Having two filenames with the same name but different cases doesn't cause any issue in a case sensitive filesystem like linux, but will cause an issue in case insensitive filesystems like Windows.

Reply:
Apologies, you're correct. Too early in the morning :)